src/assets/wise5/classroomMonitor/dataExport/strategies/LabelComponentDataExportStrategy.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '../../../common/Component';
+import { ComponentDataExportParams } from '../ComponentDataExportParams';
+import { ComponentRevisionCounter } from '../ComponentRevisionCounter';
+import { AbstractComponentDataExportStrategy } from './AbstractComponentDataExportStrategy';
+
+export class LabelComponentDataExportStrategy extends AbstractComponentDataExportStrategy {
+  maxNumLabels: number;
+
+  constructor(component: Component, additionalParams: ComponentDataExportParams) {
+    super(component, additionalParams);
+    this.maxNumLabels = 0;
+  }
+
+  export(): void {
+    this.controller.showDownloadingExportMessage();
+    const components = [{ nodeId: this.component.nodeId, componentId: this.component.id }];
+    this.dataExportService.retrieveStudentDataExport(components).then((result) => {
+      this.populateColumnNames(this.columnNames, this.columnNameToNumber);
+      let rows = [this.generateComponentHeaderRow(this.columnNames)];
+      rows = rows.concat(this.generateComponentWorkRows(this.component, this.component.nodeId));
+      this.addLabelHeaders(rows, this.maxNumLabels);
+      const fileName = this.generateExportFileName(
+        this.component.nodeId,
+        this.component.id,
+        'label'
+      );
```

Comment:
Pass in component only, handle getting nodeId, componentId, and type inside the ```generateExportFileName()``` function?
```suggestion
      const fileName = this.generateExportFileName(this.component);
```